src/components/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import { forwardRef, Dispatch } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  addTodo: (event: React.FormEvent<HTMLFormElement>) => void;
+  title: string;
+  setTitle: Dispatch<React.SetStateAction<string>>;
```

Comment:
title looks more like internal detail of the header - it makes sense to keep this state inside